java/arcs/android/devtools/DevToolsService.kt

```diff
@@ -46,6 +54,17 @@ class DevToolsService : Service() {
     }
 
     override fun onBind(intent: Intent): IBinder? {
+        scope.launch {
+            val extras = intent.extras
+            if (extras != null) {
+                storageClass = extras.getSerializable("STORAGE_CLASS") as Class<StorageService>
```

Comment:
In general, this isn't advisable (using Serializables as part of bundles/extras)... but I think for the sake of a dev tools/debug-only thing, it's probably okay.